relay-monitors/src/lib.rs

```diff
@@ -129,6 +129,7 @@ pub struct CheckIn {
     pub check_in_id: Uuid,
 
     /// Identifier of the monitor for this check-in.
+    #[serde(default)]
     pub monitor_slug: String,
```

Comment:
Slug is now no longer required since it is not required when using the `POST` API endpoint.

Theoretically you could already send the empty string as @jan-auer pointed out, so we probably don't need to handle an error condition for this missing